server/app/views/components/TextFormatter.java

```diff
@@ -51,7 +52,7 @@ public enum UrlOpenAction {
   private static final String BULLETED_ITEM = "* ";
 
   public static ImmutableList<DomContent> createLinksAndEscapeText(
```

Comment:
Would you mind javadocing this since it's public as well to be clear on the new param while you're here